### DISCUSSION TITLE
What's the proper config for playing Powder118?

### DISCUSSION THREAD

**Discussion by Znidz** [OnionUI/Onion#554] [General]
Hello

I'm a fan of the roguelike "Powder" that you've included the option to install.
Nothing seems to run though when I attempt to open Powder on the "Expert" section of the OS.
This is with all set to default.
I can't find anything on the wiki.

I'd like to say though, thanks for working on this. It's a great project. Good job guys! :)

**Comment by dwmccoy** [comment]
I have a solution.  Not sure if it's correct but it does work.  The powder launcher acts like it's looking for a file or a rom to load even though everything is contained in the executable.  Perform the following.

Edit the config.json file located in /RApp/powder118 folder of your SDCARD to look like below.  The change is made to the "extlist" value.

{
"label":"powder118",
"icon":"powder.png",
"iconsel":"powder_sel.png",
"launch":"powder",
"rompath":"./Launch",
"imgpath":"./Launch",
"useswap":1,
"shortname":0,
"hidebios":1,
"extlist":"txt"
}

#(Licensed under a deprecated CC license (https://creativecommons.org/licenses/sampling+/1.0/ sourced from http://www.zincland.com/powder/index.php?pagename=release) which allows distribution for non-commercial purposes with proper attribution (original author's name is displayed in the app). 
#"Roguelike game by Jeff Lait"
#ported by boiler

Then create a text file named powder118.txt and place it into the /RApp/powder118/Launch folder of your SDCARD with the following value.

.././powder

When you launch powder you will be presented to open the text file which in turn will start the game.  Let us know if this works for you.

**Reply by Znidz** [reply]
That did it! Thanks so much. 
Opened the game and was almost immediately killed by a kiwi :)
